main.go

```diff
@@ -52,7 +52,7 @@ func main() {
 	}()
 
 	observationWriter := observation.NewMessageWriter(kafkaProducer)
-	requestHandler := request.NewCSVHandler(s3, observationWriter)
+	requestHandler := event.NewCSVHandler(s3, observationWriter)
```

Comment:
My thinking here was that its a handler implementation specifically handles CSV files. I am happy to change it to just be handler.